pkg/rbeconfigsgen/options.go

```diff
@@ -0,0 +1,236 @@
+// Package rbeconfigsgen contains utilities to generate C++ & Java Toolchain configs for Bazel to be
+// used to run RBE builds
+package rbeconfigsgen
+
```

Comment:
Out of curiosity, is there a reason why this is a separate file but not a separate package?

Reply:
I wanted to isolate the internal structs & functions & execution logic of the config generation into its own file. The options are mostly publicly isolated and are probably most interesting to anyone who's trying to figure out how to call the Run option. This file also includes logic to apply certain default based on the OS and it'll be easier for the reader if they don't need to wade through internal structs/execution logic to find it.

I don't see a strong enough reason to create a separate package though because the Options are tightly coupled with the execution logic in that they only affect the behavior of the execution logic and I would like to create additional packages unless we have a really strong reason to.

Reply:
That makes sense.